### PR TITLE
Project documentation (tudor.balba)

### PR DESCRIPTION
**Pull request by tudorb1011**
Pull Request Overview

This pull request adds the documentation for the Pacman Game

TODO or Help Wanted
This pull request still needs to be evaluated

Author
Signed-off-by: Tudor Balba balba.tudorr@gmail.com

### CODE REVIEW

File: docs/project/index.md
Comment (alexandruradovici) [inline docs/project/index.md:1]: Please do not delete this file.

File: docs/project/2024/tudor.balba/index.md
Comment (alexandruradovici) [inline docs/project/2024/tudor.balba/index.md:1]: Please follow the proposed template for the project page.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/index.md
Comment (alexandruradovici) [reply docs/project/index.md:1]: This is jot resolved, please do not modify this file.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/index.md
Comment (tudorb1011) [reply docs/project/index.md:1]: it s not ok now? because first i saved the code there by mistake but i uploaded again

Comment (alexandruradovici) [reply docs/project/index.md:1]: The file is modified. Do not modify in any way this file.

**Pull request closed without merging**